homeassistant/components/devolo_home_control/__init__.py

```diff
@@ -25,7 +28,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the devolo account from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
-    mydevolo = configure_mydevolo(entry.data)
+    mydevolo = configure_mydevolo(dict(entry.data))
```

Comment:
Why is this being wrapped in a `dict()`?

Reply:
configure_mydevolo is called here and in the config_flow. In the config_flow, it takes user_input (which is a dict[str, Any]) and here entry.data which is MappingProxyType. So I can either allow both types or cast it. I'm fine with either way, I don't know which on is more common ...

Reply:
Checked out the code in this case, I would say allow both types.